netkan/netkan/metadata.py

```diff
@@ -121,3 +122,164 @@ def licenses(self):
             return lic
         else:
             return [lic]
+
+    class Version:
+
+        Pattern = re.compile("^(?:(?P<epoch>[0-9]+):)?(?P<version>.*)$")
```

Comment:
```suggestion
        PATTERN = re.compile("^(?:(?P<epoch>[0-9]+):)?(?P<version>.*)$")
```

I think we're going with the convention that class constants are in ALL CAPS.